pyke/kepconvert.py

```diff
@@ -121,12 +144,30 @@ def kepconvert(infile, conversion, columns, outfile=None, baddata=True,
         for colname in tqdm(colnames):
             try:
                 if colname.lower() == 'time':
-                    work.append(table.field(colname) + bjdref)
+                    if len(timeformat) > 0:

```

Comment:
For clarity, let's have `timeformat="jd"` as the default in the function definition, and then we can have `if timeformat != "jd"` here.